benches/criterion/no_op.rs

```diff
@@ -28,44 +59,52 @@ fn run_no_ops(opts: &Options, count: u64) -> Duration {
     let mut ring_opts = tokio_uring::uring_builder();
     ring_opts.setup_cqsize(opts.cq_size as _);
 
-    let mut m = Duration::ZERO;
+    tokio_uring::builder()
+        .entries(opts.sq_size as _)
+        .uring_builder(&ring_opts)
+        .start(async move {
+            let mut js = JoinSet::new();
 
-    // Run the required number of iterations
-    for _ in 0..count {
-        m += tokio_uring::builder()
-            .entries(opts.sq_size as _)
-            .uring_builder(&ring_opts)
-            .start(async move {
-                let mut js = JoinSet::new();
+            // Prepare the number of concurrent tasks this benchmark calls for.
+            // They will each be blocked until the current task yields the thread
+            // when it calls await on the first join_next below.
+            //
+            // Within each task, run through the number of no_op calls in series based on the
+            // benchmark count (accounting for the number of tasks that are being spawned).
 
-                for _ in 0..opts.iterations {
-                    js.spawn_local(tokio_uring::no_op());
-                }
+            for _ in 0..opts.concurrency {
+                let opts = opts.clone();
+                js.spawn_local(async move {
+                    // Run the required number of iterations (per the concurrency amount)
+                    for _ in 0..(count / opts.concurrency) {
```

Comment:
I'm not sure about this approach. If we just use `Throughput`, criterion will do normalization and adjustment for us.